pkg/receiver.go

```diff
@@ -221,7 +231,7 @@ func (receiver *SsfReceiverImplementation) PollEvents() ([]events.SsfEvent, erro
 			fmt.Println(err)
 		}
 	}
-	events, err := parseSsfEventSets(&ssfEventsSets.Sets)
+	events, _ := parseSsfEventSets(&ssfEventsSets.Sets)
```

Comment:
why are we ignoring the parsing errors? Or do you not expect errors to occur at this stage?